app/dashboard/views.py

```diff
@@ -218,37 +235,48 @@ def uninterested(request, bounty_id, profile_id):
     Returns:
         dict: The success key with a boolean value and accompanying error.
     """
-    page = request.GET.get('page', 1)
-    limit = request.GET.get('limit', 10)
 
-    # Get all interests for the Bounty.
-    interests = Interest.objects \
-        .filter(bounty__id=bounty_id) \
-        .select_related('profile') \
-        .order_by('created')
+    session_profile_id = request.session.get('profile_id')
+    if not session_profile_id:
+        return JsonResponse(
+            {'error': 'You must be authenticated!'},
+            status=401)
 
-    paginator = Paginator(interests, limit)
     try:
-        interests = paginator.page(page)
-    except PageNotAnInteger:
-        interests = paginator.page(1)
-    except EmptyPage:
-        return JsonResponse([])
-
-    interests_data = []
-    for interest in interests:
-        interest_data = ProfileSerializer(interest.profile).data
-        interests_data.append(interest_data)
-
-    if request.is_ajax():
-        return JsonResponse(json.dumps(interests_data), safe=False)
-
-    return JsonResponse({
-        'paginator': {
-            'num_pages': interests.paginator.num_pages,
-        },
-        'data': interests_data,
-    })
+        bounty = Bounty.objects.get(pk=bounty_id)
+    except Bounty.DoesNotExist:
+        return JsonResponse({'errors': ['Bounty doesn\'t exist!']},
+                            status=401)
```

Comment:
Minor thing but if bounty doesn't exist this should be a 404, right?

Reply:
Googled it up and came across [this](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.5)

Reply:
Not sure where/why this showed up here, but I didn't add that xD It was from existing code that changed lines.  Agreed, though!

Reply:
@mbeacom sure we believe you :P

Reply:
@thelostone-mc 😛  https://github.com/gitcoinco/web/blob/master/app/dashboard/views.py#L199

Reply:
@thelostone-mc hmm you believe this or not? Not sure if I'm convinced 😂🤣